packages/core/src/view/handler/SelectionHandler.ts

```diff
@@ -208,7 +208,7 @@ class SelectionHandler implements GraphPlugin {
    * Specifies if other cells should be used for snapping the right, center or
    * left side of the current selection. Default is false.
    */
-  guidesEnabled = false;
+  guidesEnabled = true;
```

Comment:
If `guidesEnabled` should be `true` by default, please update the comments above. :)